librocksdb_sys/crocksdb/c.cc

```diff
@@ -3331,6 +3331,14 @@ void crocksdb_lru_cache_options_set_high_pri_pool_ratio(
   opt->rep.high_pri_pool_ratio = high_pri_pool_ratio;
 }
 
+void crocksdb_lru_cache_options_set_no_dump_allocator(
+    crocksdb_lru_cache_options_t* opt) {
+    std::shared_ptr<rocksdb::MemoryAllocator> allocator;
+    rocksdb::JemallocAllocatorOptions options;
```

Comment:
I don't konw if we need set JemallocAllocatorOptions @yiwu-arbug

Reply:
Default options for `JemallocAllocatorOptions` is fine.